Sources/Nimble/Matchers/BeginWithPrefix.swift

```diff
@@ -0,0 +1,41 @@
+import Foundation
+
+/// A Nimble matcher that succeeds when the exepected sequence is a prefix of the actual sequence.
```

Comment:
```suggestion
/// A Nimble matcher that succeeds when the exepected sequence is a prefix of the actual sequence.
```

Foundation types are not used.